flutter-for-android.md

```diff
@@ -1440,6 +1440,8 @@ children.
   }
 {% endprettify %}
 
+To learn more in detail how to build LinearLayout, see this [Article](https://medium.com/@burhanrashid52/flutter-for-android-developers-how-to-design-linearlayout-in-flutter-5d819c0ddf1a).
```

Comment:
Can you reword a bit?

To learn more about building linear layouts, see the medium article [Flutter For Android Developers : How to design LinearLayout in Flutter?](https://proandroiddev.com/flutter-for-android-developers-how-to-design-linearlayout-in-flutter-5d819c0ddf1a).